core/engine/src/builtins/temporal/plain_month_day/mod.rs

```diff
@@ -33,6 +43,123 @@ impl PlainMonthDay {
     }
 }
 
+// ==== `Temporal.PlainMonthDay` static Methods ====
+impl PlainMonthDay {
+    // 10.2.2 Temporal.PlainMonthDay.from ( item [ , options ] )
+    fn from(_: &JsValue, args: &[JsValue], context: &mut Context) -> JsResult<JsValue> {
+        let options = get_options_object(args.get_or_undefined(1))?;
+        let item = args.get_or_undefined(0);
+        let inner = if item.is_object() {
```

Comment:
suggestion: extract into `to_temporal_month_day` function

Technically, `ToTemporalMonthDay` is only called twice by by `from` and `equals`, so I could see the argument to not extract it. But having a to-X method implementation would be consistent with all the other built-ins and hopefully make everything a bit more readable.